### ISSUE TITLE
Adding Shortcut Bug

### ISSUE THREAD

**Issue by username227** [LSPosed/LSPosed#1330]
### Steps to reproduce/复现步骤

I added Nova Launcher. 
I followed prompts to install a parasitic manager shortcut on the new launcher.
The shortcut wasn't added to the new launcher but the old launcher.

### Expected behaviour/预期行为

It should add the shortcut to the new launcher.

### Actual behaviour/实际行为

It adds it back to the old launcher.

### Xposed Module List/Xposed 模块列表

```shell
GravityBox
Xprivacylua
HideMyApplist
```

### Magisk Module List/Magisk 模块列表

```shell
Busybox
Viper4Android fx
Riru
Lsposed-riru
Audio Modification Library
Magiskhide Props Config
```

### LSPosed version/LSPosed 版本

1.6.2

### Android version/安卓版本

10

### Magisk version/Magisk 版本

23.0

### Riru version/Riru 版本

26.1.3

### Version requirement/版本要求

- [X] I am using latest debug CI version of LSPosed and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志

[LSPosed_2021-10-25T20_58_37.075.zip](https://github.com/LSPosed/LSPosed/files/7414292/LSPosed_2021-10-25T20_58_37.075.zip)

**Comment by zzzsyyy** [comment]
> > @Howard20181 Well, it was added to the non-default one. Is there any way to manually create a shortcut for this? Thanks.
> 
> Remove the shortcut from the non-default launcher and then reboot. This process is controlled by the system, consider giving feedback to the ROM developer.

遇到同样的问题，然而在其他软件里支持这一点，（如kiwi browser
*wantfix*
![Screenshot_20211030-181356_Nova7](https://user-images.githubusercontent.com/59917878/139529043-88554bc9-a90f-4eb2-9ce5-1d41809d12f9.jpg)

**Comment by Howard20181** [comment]
> > > @Howard20181 Well, it was added to the non-default one. Is there any way to manually create a shortcut for this? Thanks.
> > 
> > 
> > Remove the shortcut from the non-default launcher and then reboot. This process is controlled by the system, consider giving feedback to the ROM developer.
> 
> 遇到同样的问题，然而在其他软件里支持这一点，（如kiwi browser _wantfix_ ![Screenshot_20211030-181356_Nova7](https://user-images.githubusercontent.com/59917878/139529043-88554bc9-a90f-4eb2-9ce5-1d41809d12f9.jpg)

在弹出来这个框之后就是系统负责的了，没有办法指定加到哪个桌面。默认会加到默认桌面，你可以试试删掉已有图标，重新设置默认桌面，再重新触发加快捷方式

**Comment by yujincheng08** [comment]
我们做的事是【问系统有没有这个图标】，没有就【请求系统帮我们加一个】。作用在哪个启动器我们完全不可控。要么是你系统有魔改，要么是 AOSP 本身就有的问题，要么是启动器不会自动显示系统已添加的图标。反正跟我们没有关系。

**Comment by furopi** [comment]
So after all it stopped working 😞
I finally managed to get the manager up and running again. It was hours of work... The solution I found was to use the app "Unnotification" to make the notification visible again and then restore the shortcut from the settings of the manager. Unnotification was once in the Play Store, but sadly isn't anymore... But the APK can be found on sites like apkpure! Hope this helps at least somebody! 

Cheers! 🫡
Florian
